prep_data_school.py

```diff
@@ -0,0 +1,152 @@
+import argparse
+import os.path
+import sys
+from pathlib import PurePath, PurePosixPath
+from posixpath import abspath, dirname
+from typing import List, Optional
+
+import pandas as pd
+import unidecode
+from numpy import dtype, str0
+
+
+def remove_characters(input_text: str, chars_to_remove: List[str]) -> str:
+    """Method for data preparation
+
+    Parameters
+    ----------
+    input_text from df
+    chars_to_remove in input_text
+    """
+
+    for char_to_remove in chars_to_remove:
+        input_text = input_text.replace(char_to_remove, "")
+    return input_text.strip()
+
+
+def prep_data(path_input: str, path_output: Optional[str] = None):
+    """Method for data preparation
+
+    Parameters
+    ----------
+    path_input
+        Input csv path
```

Comment:
Tutaj warto doprecyzować, że chodzi o `directory`/`folder` bo z opisu zrozumiałbym, że to ma być ścieżka bezpośrednio do `.csv`